pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -61,6 +61,10 @@ type BuildRunStatus struct {
 	// +optional
 	LatestTaskRunRef *string `json:"latestTaskRunRef,omitempty"`
 
+	// ServiceAccountName is the name of the serviceAccount used to run the build.
+	// +optional
+	ServiceAccountName *string `json:"serviceAccountName,omitempty"`
```

Comment:
Please be careful the property `spec.serviceAccount.generate`, if it is true, the service account will be deleted in the end.

So I am thinking, if we can enhance some document like:
https://github.com/shipwright-io/build/blob/master/docs/buildrun.md#defining-the-serviceaccount
or
https://github.com/shipwright-io/build/blob/master/docs/buildrun.md#buildrun-status

To tell the serviceaccount name will be shown in the buildrun status.

Reply:
I will add this information in the *note* section there. Thanks @zhangtbj